website/blog/2022-06-15-version-069.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Announcing React Native 0.68
+authors: [fortmarek, cortinico]
+tags: [announcement, release]
+---
+
+# Announcing 0.69
+
+Hey all, 👋
+
+We are excited to release a new version of React Native, 0.69.0. This version comes with several improvements for the New Architecture of React Native and new features: React 18 support & bundled Hermes. Read on to learn more!
+
+### Sections
+
+- [React 18](/blog/2022/06/15/version-069#react-18)
+- [Bundled Hermes](/blog/2022/06/15/version-069#bundled-hermes)
+- [Highlights of 0.69](/blog/2022/06/15/version-069#highlights-of-068)
+
+<!--truncate-->
+
+## React 18
+
+We are excited to share with you that React Native 0.69 is the first release to support React 18. React 18 has brought [lots of improvements](https://reactjs.org/blog/2022/03/29/react-v18.html), like new hooks such as `useId`. Additionally, React 18 includes new concurrency features such as `useTransition` or full Suspense support.
+
+On React Native 0.69, React 18 is enabled by default. However, if you have not migrated to the New Architecture, you will only be able to leverage the features that don't use concurrent rendering and concurrent features. New Architecture has been built with concurrent rendering in mind but we cannot add that support for the old architecture.
+
+You can learn more about the React 18 support in React Native [here](https://reactnative.dev/docs/next/react-18-and-react-native).
+
+## Bundled Hermes
+
+Before this release, Hermes and React Native were released separately. That led to confusion on which version of Hermes is compatible with which version of React Native. To remedy this issue, starting with React Native 0.69 we will be shipping a compatible version of Hermes alongside React Native. Making this change will make using Hermes in React Native much more stable.
+
+Using the proper version of Hermes is handled by React Native, just make sure to follow the steps in the [upgrade helper](https://react-native-community.github.io/upgrade-helper/?from=0.68.2&to=0.69.0) to ensure the integration works as intended. If you don't have Hermes enabled already, you can follow the steps [here](/docs/hermes) to do so. While we will continue supporting other JavaScript engines, we recommend everyone to migrate to Hermes to have the best experience and to make sure we can support you better.
```

Comment:
currently depends on https://react-native-community.github.io/upgrade-helper/?from=0.68.2&to=0.69.0 going live first too right (0.69.0-rc.6 re-direct atm on the btn click)

Reply:
yes, that's although a safe assumption (this blogpost should only be published once `0.69.0` is actually released. The upgrade helper is updated automatically as soon as a new RN version tag is pushed.